Add unit test to validate API server functionality

MichaelKim20 opened the pull request:
The unit test code allows the API server to be started and shut down.
This will keep API server functions in good quality continuously.

Related to issue #42

TrustHenry commented on tests/SampleData.test.ts:
```diff
@@ -1,3 +1,15 @@
+/*******************************************************************************
+
+    The sample block data for the test.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+ *******************************************************************************/
```
This part is the same as the committed contents of bf254477

MichaelKim20 replied:
Oh, it's my mistake. I fixed it.

MichaelKim20 committed:
Detach required variables in multiple locations

MichaelKim20 committed:
Add unit test to validate API server functionality

The unit test code allows the API server to be started and shut down.
Add package 'axios' and 'urijs' required for testing.
`axios` is promise based HTTP client for the browser and node.js.
`urijs`is a library for working with URLs.

TrustHenry reviewed (approved):
LGTM
